### PR TITLE
Add glossary of terms

### PR DESCRIPTION
**Pull request by reub-the-cube**
- Refactor pages to have spaced out order (allow easier insertion)
- Add sub-nav for Intro to DECS (was get started)

### CODE REVIEW

File: documentation/code-repos.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: sub-navigation
-order: 4
+order: 40
```
Comment (reub-the-cube) [inline documentation/code-repos.md:3]: Order should be the same but I've left gaps to make it easier to insert pages

File: documentation/get-started.md [hunk 2]
```diff
@@ -1,14 +1,13 @@
 ---
```
Comment (reub-the-cube) [inline documentation/get-started.md:1]: Moved this page into a collection (first page of the collection)

File: documentation/get-started/domain-map.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+order: 30
+title: Domain map
+description: Set of statements about how domain entities relate to each other.
+---
+
+## Direct relationships
+- A case is for a single case type
+- A case type is owned by a single business unit
+- A business unit can own multiple case types
+- A case consists of multiple stages
```
Comment (CaseyProsser) [inline documentation/get-started/domain-map.md:12]: I think MTS case type has only a single stage.

File: documentation/get-started/domain-map.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+- A user can belong to multiple user groups
+- A user group is assigned to a single case type
```
Comment (CaseyProsser) [inline documentation/get-started/domain-map.md:14]: I think user groups are assigned to business units, not case types (I am interpreting this from the fact that being in a DCU team gives you read access to all 3 DCU case types). Either that or user groups can be assigned to multiple case types.

File: documentation/get-started/domain-map.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+- A user group can be assigned to multiple stages of a case type
+- A user group has a single workstack
+
+## Indirect relationships
+- A user can see multiple workstacks (they can belong to many user groups)
+- A user cannot see cases that are in the same stage across case types in a single view
```
Comment (CaseyProsser) [inline documentation/get-started/domain-map.md:20]: I think this is saying that workstacks cannot contain multiple case types? Which would be incorrect. DCU workstacks routinely have multiple case types in 1 workstack view.

**Comment by nickmcmahon01** [comment]
Lets organise a session, we have existing UL stuff, so we should use that as a starting pount

File: documentation/get-started/domain-map.md (hunk 3 above)
Comment (reub-the-cube) [reply documentation/get-started/domain-map.md:12]: I've not come across MTS, but multiple means 'one or more', so I think it's OK?

File: documentation/get-started/domain-map.md (hunk 4 above)
Comment (reub-the-cube) [reply documentation/get-started/domain-map.md:14]: OK, this is great - I can definitely have not understood correctly!

File: documentation/get-started/domain-map.md (hunk 5 above)
Comment (reub-the-cube) [reply documentation/get-started/domain-map.md:20]: Yes, that dependency would have been a limitation from not being able to have multiple case types for a single user group (but I may have got that wrong as above).

**Pull request closed without merging**